sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/DocumentClassifierDetails.java

```diff
@@ -0,0 +1,190 @@
+    public DocumentClassifierDetails() {}
+
+    /**
+     * Get the classifierId property: Unique document classifier name.
+     *
+     * @return the classifierId value.
+     */
+    public String getClassifierId() {
+        return this.classifierId;
+    }
+
+    /**
+     * Set the classifierId property: Unique document classifier name.
+     *
+     * @param classifierId the classifierId value to set.
+     */
+    void setClassifierId(String classifierId) {
+        this.classifierId = classifierId;
+    }
+
+    /**
+     * Get the description property: Document classifier description.
+     *
+     * @return the description value.
+     */
+    public String getDescription() {
+        return this.description;
+    }
+
+    /**
+     * Set the description property: Document classifier description.
+     *
+     * @param description the description value to set.
+     */
+    void setDescription(String description) {
+        this.description = description;
+    }
+
+    /**
+     * Get the createdDateTime property: Date and time (UTC) when the document classifier was created.
+     *
+     * @return the createdDateTime value.
+     */
+    public OffsetDateTime getCreatedOn() {
+        return this.createdDateTime;
+    }
+
+    /**
+     * Set the createdDateTime property: Date and time (UTC) when the document classifier was created.
+     *
+     * @param createdDateTime the createdDateTime value to set.
+     */
+    void setCreatedDateTime(OffsetDateTime createdDateTime) {
+        this.createdDateTime = createdDateTime;
+    }
+
+    /**
+     * Get the expirationDateTime property: Date and time (UTC) when the document classifier will expire.
+     *
+     * @return the expirationDateTime value.
+     */
+    public OffsetDateTime getExpiresOn() {
+        return this.expirationDateTime;
+    }
+
+    /**
+     * Set the expirationDateTime property: Date and time (UTC) when the document classifier will expire.
+     *
+     * @param expirationDateTime the expirationDateTime value to set.
+     */
+    void setExpirationDateTime(OffsetDateTime expirationDateTime) {
+        this.expirationDateTime = expirationDateTime;
+    }
+
+    /**
+     * Get the apiVersion property: API version used to create this document classifier.
+     *
+     * @return the apiVersion value.
+     */
+    public String getApiVersion() {
+        return this.apiVersion;
+    }
+
+    /**
+     * Set the apiVersion property: API version used to create this document classifier.
+     *
+     * @param apiVersion the apiVersion value to set.
+     */
+    void setApiVersion(String apiVersion) {
+        this.apiVersion = apiVersion;
+    }
+
+    /**
+     * Get the docTypes property: List of document types to classify against.
+     *
+     * @return the docTypes value.
+     */
+    public Map<String, com.azure.ai.formrecognizer.documentanalysis.implementation.models.ClassifierDocumentTypeDetails> getDocTypes() {
```

Comment:
Is this class also a public explored class?

Reply:
good catch!